xtheadmaee.adoc

```diff
@@ -0,0 +1,34 @@
+[#xtheadmaee]
+== Extend MMU address attribute
+
+[NOTE,caption=Frozen]
+The `XTheadMaee` extension is `stable`.
+
+The `XTheadMaee` ISA extension provides a way to configure page attributes of addresses when virtual addresses are translated into physical addresses by extended page attributesin page table entries (PTEs).
+
+Extension version: 1.0.
+
+The table below describes the page attributes corresponding to each memory type:
+
+[cols="^3,^3,12,18",stripes=even,options="header"]
+|===
+| Memory type            | SO | C | B
+| Cacheable memory       | 0  | 1 | 1
+| Non-cacheable memory   | 0  | 0 | 1
+| Bufferable device      | 1  | 0 | 1
+| Non-bufferable device  | 1  | 0 | 0
+|===
+
+The table below describes extend page attributes in PTE:
+[cols="^3,^3,12",stripes=even,options="header"]
+|===
+| Fields  | Bit | Meaning
+| SO      | 63  | Strong order
+| C       | 62  | Cacheable
+| B       | 61  | Bufferable
+| T       | 60  | Trustable
+|===
+
+XTheadMaee depends on the value of the MAEE field in extended register MXSTATUS. If the MAEE field is 1, page attributes of addresses are determined by extended page attributes in corresponding PTEs. If the MAEE field is 0, page attributes of addresses are determined by the sysmap.h file.
```

Comment:
It is not defined (or referenced) what `MXSTATUS` is (it is a custom CSR with a certain number).
Also, we should probably give that CSR a prefix (e.g. `TH_`) like we do with instruction mnemonics to prevent name collisions with standard CSR names.

Reply:
OK. Yes, we can name it to TH_MXSTATUS. It's number is 0x7C0.